pages/event-indexing.md

```diff
@@ -0,0 +1,22 @@
+# Indexing ABCI events
+
+The [`pd`](./node/pd) software emits ABCI events while processing blocks. By default,
+these blocks are stored in CometBFT's key-value database locally, but node operators
+can opt-in to writing the events to an external PostgreSQL database.
+Furthermore, the [`pindexer`](./event-indexing/pindexer.md) software can be used to take these raw ABCI events,
+and produce Penumbra-specific "app views" with rich formatted data.
+
+## Motivation for event indexer
+
+Any application that requires detailed information about Penumbra chain state
+needs access to an event database, in order to display information to a user.
+This is because Penumbra is private by default, so the majority of chain state
+is encrypted, and therefore not legible to naive external tooling.
```

Comment:
suggestion: I don't think this is quite the motivation.

While a lot of the state is indeed private, when this is true, we also can't emit any events relating to that state either!
The concern is more so that we want to avoid burdening nodes with the task of maintaining historical data that isn't needed to process new blocks.
Because of this, we use "events" as a means for the external world to track historical happenings on the chain, and maintain heavier indices of non-consensus critical state.

Reply:
Thanks, this is exactly the kind of real-talk I was hoping to solicit with the review request. #64 has some suggestions to make the docs more navigable to newcomers, and much of the explication I want to add here better belongs as part of that effort.